Changelog.md

```diff
@@ -13,6 +13,8 @@ Breaking Changes:
  * Code Generator: Use ``revert`` with error signature ``Panic(uint256)`` and error codes instead of invalid opcode on failing assertions.
  * Type System: Explicit conversions from literals to integer type is as strict as implicit conversions.
  * Type System: Explicit conversions from literals to enums are only allowed if the value fits in the enum.
+ * Command Line Interface: Remove the legacy ``--ast-json`` option. Only the ``--ast-compact-json`` option is supported now.
+ * Standard JSON: Remove the ``legacyAST`` option.
```

Comment:
I did not sort these, as they will be sorted before the release anyway.

Reply:
I guess it is fine, but it would be great to keep it as sorted as possible, so it is less work for release :)

Reply:
I kept it sorted on other PRs but it quickly fell out of being sorted with other PR so I don't bother with it anymore :)

Reply:
Fair enough